File: APIconvenios/DTOs/Convenios/ConvenioUltimoDto.cs
using APIconvenios.Common.Enums;

namespace APIconvenios.DTOs.Convenios
{
    public class ConvenioUltimoDto
    {
        public int Id { get; set; }
        public string? Titulo { get; set; }
        public string ConvenioType { get; set; } = string.Empty;
        public string? NombreEmpresa { get; set; }
        public EstadoConvenio Estado { get; set; }
    }

    public class UltimosConveniosDto
    {
        public List<ConvenioUltimoDto> ConveniosMarcos { get; set; } = new();
        public List<ConvenioUltimoDto> ConveniosEspecificos { get; set; } = new();
    }
}

File: UserInterface/src/Types/Convenios/ConvenioUltimoDto.ts
import type { EstadoConvenio } from '@/Types/Enums/Enums'

export interface ConvenioUltimoDto {
  id: number
  titulo: string | null
  convenioType: 'marco' | 'especifico'
  nombreEmpresa: string | null
  estado: EstadoConvenio
}

export interface UltimosConveniosDto {
  conveniosMarcos: ConvenioUltimoDto[]
  conveniosEspecificos: ConvenioUltimoDto[]
}
